js/pages/iptt_report/components/sidebar/sidebar.js

```diff
@@ -1,21 +1,24 @@
 import React, { useEffect, useState } from 'react';
 import IPTTFilterForm from './filterForm';
 
-export default ({bottomScrolling, extend}) => {
+export default ({bottomScrolling, extend, tableHeight}) => {
+    const [startScroll, setStartScroll] = useState(0);
```

Comment:
The general comments below are useful but additional comments on what some of these specific variables represent would be helpful, particularly bottomScrolling and startScroll.

Reply:
I added a lot more comments, definitions, and an example. Let me know if it makes it easier to understand.

Reply:
The additions were perfect, thanks!